root/src/root-application.js

```diff
@@ -9,6 +10,10 @@ const showLoading = () => {
   }
 };
 
+window.addEventListener("single-spa:first-mount", () => {
+  console.info(`${appName} ${appVersion} (${process.env.GIT_SHA}).`);
```

Comment:
This `console.info` could just be inside this module and will be printed when the module is initialised as it doesn't actually need to wait for the mount to happen, but inside the listener is fine too.